### PR TITLE
fix(deploy-tooling): hide stack trace on error

### PR DESCRIPTION
**Pull request by mikicvi-SAP**
Fixes: Additional bug from #1758 where stack trace from axios was still shown.
- Fix for internal issue no: 27150

### CODE REVIEW

File: packages/deploy-tooling/test/unit/base/validate.test.ts [hunk 1]
```diff
@@ -480,7 +480,7 @@ describe('deploy-test validation', () => {
             }
         );
     });
-    describe('Validate error does not show full stack trace', () => {
+    describe.only('Validate error does not show full stack trace', () => {
```
Comment (devinea) [inline packages/deploy-tooling/test/unit/base/validate.test.ts:483]:
```suggestion
    describe('Validate error does not show full stack trace', () => {
```

Comment (mikicvi-SAP) [reply packages/deploy-tooling/test/unit/base/validate.test.ts]: Thank you Austin, addressed in [c824d87](https://github.com/SAP/open-ux-tools/pull/1809/commits/c824d875528b1c9916749f8bbe0982c31b27a998)

**Review by devinea — APPROVED** [review]
Code changes and test updates look appropriate. 
Changeset ✅

**Review by longieirl — APPROVED** [review]
Changes look good.
Tests are appropiate.
Changeset present.

**Review by slin-sap — APPROVED** [review]
- changeset is present with patch bump
- fix looks good to me

**Pull request merged**